### ISSUE TITLE
Export of CT-data

### ISSUE THREAD

**Issue by maikeros** [OpenGATE/Gate#623]
Hello everyone,

I am new using Gate for my master thesis and I am trying to simulate CT-scans of different phantoms to compare it with real CT-scans and reconstruct the data with different algorithms. I used the example of a CT-Simulation from Gate first and I get the CT ra=
w data in form of .dat and .root files. How can I work with the .dat data to create a CT-image? I tried to read the .dat file with python, but then I need the dimensions of the CT-image and I don't know, where to get them, because they are not the same as the dimensions of the geometry in Gate.
I also tried to create Interfiles as an output and used the code from the Gate documentation, but that only gave me an error message.
Additionally, I got the dose deposition/NumberofHits etc. as an raw and mhd file, but I can also not create a CT-image only from that.

So basically my question is, what is the best way to export the CT-data from Gate and create and recontruct an CT-image out of that. Maybe it's a stupid question, but I investigated the root data file for some time now and also tried to convert the files with python, but I did not come to a solution yet.

I hope, that you can help me! Thank you and kind regards!

Maike
